packages/stats/gbstats/bayesian/tests.py

```diff
@@ -158,6 +144,23 @@ def compute_result(self):
         if self._has_zero_variance():
             return self._default_output(ZERO_NEGATIVE_VARIANCE_MESSAGE)
 
+        # rescale prior if needed
+        scaled_prior_effect = self.config.prior_effect
+        if self.relative and self.config == "absolute":
+            scaled_prior_effect = GaussianPrior(
+                self.config.prior_effect.mean / abs(self.stat_a.unadjusted_mean),
+                self.config.prior_effect.variance / pow(self.stat_a.unadjusted_mean, 2),
+                self.config.prior_effect.proper,
+            )
+        elif not self.relative and self.config.prior_type == "relative":
+            if self.config.prior_effect.proper and self.stat_a.unadjusted_mean <= 0:
```

Comment:
couldn't we scale by absolute value here if unadjusted_mean < 0?  We take absolute value when scaling below for the positive case.  So we should either permit unajudsted_mean < 0, or remove the abs() call below.

Reply:
Sure, we just have other problems with negative metrics but I think it should be fine here.